Core/Core/InstUI/Views/Divider.swift

```diff
@@ -18,13 +18,11 @@
 
 import SwiftUI
 
-public struct CoreDividerView: View {
-    private static let dividerColor = Color(UIColor { traits in
-        traits.isDarkInterface ? .oxford : .tiara
-    })
-
-    public var body: some View {
-        Divider().overlay(Self.dividerColor)
+public extension InstUI {
+    struct Divider: View {
+        public var body: some View {
+            Divider().overlay(Color.borderMedium)
```

Comment:
This one confuses me a bit, I don't see clearly why doesn't it refer to it's own init.
Maybe it would be worth to be explicit in these cases with `SwiftUI.Divider()`

Reply:
It does refer to its own init! It got broken when I moved this to the InstUI namespace and renamed from CoreDivider to Divider.